common/src/bitset.rs

```diff
@@ -144,6 +262,18 @@ impl BitSet {
         }
     }
 
+    /// Create a new `BitSet` that may contain elements
+    /// within `[0, max_val)`.
+    pub fn with_max_value_and_filled(max_value: u32) -> BitSet {
```

Comment:
`with_all_docs(max_value: u32)`?

Reply:
"full" would have the benefit of sticking with the TinySet naming.

Reply:
I would change the remaining bits (above max_doc) of the  last TinyBitSet to 0.

Reply:
full seems like a good name to me

> I would change the remaining bits (above max_doc) of the last TinyBitSet to 0.

I considered that, but they aren't supposed to be used anyway?